package.json

```diff
@@ -63,6 +63,9 @@
     "neverBuiltDependencies": [
       "sku"
     ],
+    "overrides": {
+      "vite": "^4.3.0-beta.1"
```

Comment:
[Beta 2 is out!](https://github.com/vitejs/vite/blob/v4.3.0-beta.2/packages/vite/CHANGELOG.md#430-beta2-2023-04-05)